packages/palette/src/elements/Text/tokens/v3.ts

```diff
@@ -0,0 +1,41 @@
+import { TEXT_FONTS, TextTreatment } from "./v2"
+
+export const V3_TEXT_FONTS = { sans: TEXT_FONTS.sans }
+
+export const V3_TEXT_TREATMENTS = ["xxl", "xl", "lg", "md", "sm", "xs"] as const
+
+export const V3_TEXT_VARIANTS: Record<
+  typeof V3_TEXT_TREATMENTS[number],
+  TextTreatment
+> = {
+  xxl: {
+    fontSize: "50px",
+    lineHeight: "54px",
+    letterSpacing: "-0.02em",
+  },
+  xl: {
+    fontSize: "34px",
+    lineHeight: "38px",
+    letterSpacing: "-0.02em",
+  },
+  lg: {
+    fontSize: "24px",
+    lineHeight: "28px",
+    letterSpacing: "-0.02em",
+  },
+  md: {
+    fontSize: "16px",
+    lineHeight: "20px",
+  },
+  sm: {
+    fontSize: "16px",
+    lineHeight: "24px",
+  },
+  xs: {
+    fontSize: "13px",
+    lineHeight: "18px",
+  },
+}
```

Comment:
Instead of having a whole zoo of font-sizes, line-heights, letter-spacings; just keeping it simple and these are the only type treatments allowed. You can step outside of this by passing raw values.

We only use the tokenized sizes/line-heights in a few places in Force, so I'm thinking I can inline those values in force then do the same thing with the old text tokens which should simplify things a bit.